setup.py

```diff
@@ -70,7 +70,7 @@ def build_extension(self, ext: CMakeExtension) -> None:
 
         # Using this requires trailing slash for auto-detection & inclusion of
         # auxiliary "native" libs
-        build_type = os.environ.get("DFTRACER_BUILD_TYPE", "Release")
+        build_type = os.environ.get("DFTRACER_BUILD_TYPE", "Debug") # Setting this to release causes memory issues with GCC-13.
```

Comment:
@hariharan-devarajan : if `RelWithDebInfo` works, that might be better here?

Reply:
I agree with @pramodk

Reply:
No it doesn't work with `RelWithDebInfo . Currently, only Debug works.

I think the issue is that string literals are getting optimized and not correctly allocated. I need to investigate the issue more.

Reply:
got it, I will approve this request